### PR TITLE
hammer commands updated

### PR DESCRIPTION
**Pull request by pondrejk**
additional changes to hammer subcommand

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_hammer.py -k test_positive_all_options

**Review by synkd — COMMENTED** [review]
Looks like PRT failed because of the issue addressed by https://github.com/SatelliteQE/robottelo/pull/10983. Since that is now merged, I'll rerun PRT.

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_hammer.py -k test_positive_all_options

**Review by JacobCallahan — APPROVED** [review]

File: tests/foreman/data/hammer_commands.json [hunk 1]
```diff
@@ -54786,4 +54829,4 @@
       ]
     }
   ]
-}
+}
```
Comment (JacobCallahan) [inline tests/foreman/data/hammer_commands.json:54832]:
```suggestion
}

```

**Comment by Gauravtalreja1** [comment]
> Hello, I created the new command tree using the hammer_command_tree.py script against 6.13 to fix tests there, I didn't anticipate there will be more changes in stream, hence the prt failure.
> 
> @JacobCallahan @Gauravtalreja1 Would you mind merging this so that 6.13 pr gets created by autocherrypick + I promise I will make a pr with additional fixes just for the master. Or I could update this pr for stream and will create a 6.13 pr manually.

@pondrejk I'd suggest changing the target branch of this PR to 6.13.z, instead of merging this PR in the master without PRT results, and opening a separate PR for stream later(and which I'm not sure is required as the stream will have some new hammer changes regularly with each snap)

**Comment by pondrejk** [comment]
changing target branch would also include other master commits to this pr, therefore closing in favour of https://github.com/SatelliteQE/robottelo/pull/11003

**Pull request closed without merging**